package.json

```diff
@@ -86,11 +86,11 @@
     "@babel/core": "7.3.4",
     "babel-core": "7.0.0-bridge.0",
     "babel-preset-cozy-app": "1.5.1",
-    "cozy-client": "6.20.0",
+    "cozy-client": "6.24.0",
     "cozy-device-helper": "1.7.1",
-    "cozy-harvest-lib": "0.41.0",
+    "cozy-harvest-lib": "0.41.0-beta.7",
```

Comment:
We may don't need Harvest to be in beta, realtime should be a peerDeps in recent versions

Reply:
to be in dependencies you mean ?

Reply:
I mean we don't need to update the realtime dependency in Harvest for here to work correctly in Cozy-Bar

Reply:
OK so if I understand correctly, we can remove cozy-harvest-lib from the dependencies, and just have it in devDep/peerDep ?

Reply:
I was saying that we don't have to update the harvest lib here to use the new realtime package, which is true, but since this is a breaking change, we need the changes on Harvest side. So, my bad, I made noise for nothing^^